src/main/java/Du/TaskList.java

```diff
@@ -0,0 +1,108 @@
+package Du;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    protected static ArrayList<Task> list_of_tasks = new ArrayList<>();
```

Comment:
can consider changing the name to listOfTasks or taskList instead to follow the camelCase coding standard! :)